src/playground/services/invoices/update/endpoint.clj

```diff
@@ -13,10 +14,8 @@
   (let [db     (->> db :pool (hash-map :datasource))
         update (-> (logic/to-update id amount (java.util.UUID/randomUUID))
                    (h/format))
-        fetch  (h/format logic/to-query)
+        fetch  (h/format (retrieve-all.logic/query-all))
         _      (jdbc/execute! db update)
-        result (-> (jdbc/query db fetch)
-                   (logic/to-serialize))]
-
-    {:status 200
-     :body   "OK"}))
+        result (jdbc/query db fetch)]
+    
+    {:result result :confirmation (str "The entry " id " has been updated.")}))
```

Comment:
what do you think of returning such a map? is it too obscure?

Reply:
Not clean yep.

Will leave another comment with the answer